be/src/exec/es_http_scan_node.cpp

```diff
@@ -92,14 +95,20 @@ Status EsHttpScanNode::prepare(RuntimeState* state) {
 // build predicate
 Status EsHttpScanNode::build_conjuncts_list() {
     Status status = Status::OK();
+    _conjunct_to_predicate.resize(_conjunct_ctxs.size());
```

Comment:
maybe the `_conjunct_to_predicate` should be local var and may be bool is ok?

Reply:
> maybe the `_conjunct_to_predicate` should be local var and may be bool is ok?

_conjunct_to_predicate record index and useful